iocage/cli/activate.py

```diff
@@ -34,9 +36,22 @@
 @click.argument("zpool")
 def cli(zpool):
     """Calls ZFS set to change the property org.freebsd.ioc:active to yes."""
-    ioc.IOCage(activate=True).activate(zpool)
+    logger = Logger.Logger()
+    zfs = libzfs.ZFS(history=True, history_prefix="<iocage>")
+    iocage_pool = None
 
-    ioc_common.logit({
-        "level"  : "INFO",
-        "message": f"ZFS pool '{zpool}' successfully activated."
-    })
+    for pool in zfs.pools:
+      if pool.name == zpool:
+        iocage_pool = pool
+
+    if iocage_pool is None:
+      logger.error(f"ZFS pool '{zpool}' not found")
+      sys.exit(1)
+
+    try:
+      datasets = Datasets.Datasets(pool=iocage_pool, zfs=zfs, logger=logger)
+      datasets.activate()
+      logger.log(f"ZFS pool '{zpool}' activated")
+    except:
+      raise
+      sys.exit(1)
```

Comment:
I want to get rid of the raise here. This was left over from debugging and I wanted to think about a proper message for the user. You probably found raise followed by an exit at other places as well. I'm following the plan that the library exits with 0 or throws an exception and the cli never raises, but prints an error and exits with status code 1.